other-docs/getting-started/README.md

```diff
@@ -70,8 +70,6 @@ For local development, you'll also need to add a local server to your developmen
 
 Local Server requires [Docker Desktop for Mac or Windows](https://www.docker.com/products/docker-desktop), or Docker Engine on Linux.
 
-If you are unable to run Docker Desktop or Docker Engine on your machine you can use the alternative local development environment based on [Chassis](https://chassis.io/). Chassis requires [Vagrant](https://www.vagrantup.com/) and [VirtualBox](https://www.virtualbox.org/).
```

Comment:
As this is the version where we tell people chassis is deprecated; rather than remove mentions of chassis, perhaps we should add a note everywhere it is mentioned to say it is now deprecated and link to the chassis page where we add a description of why it is deprecated and when it will be removed.

If it makes sense in the flow of the document, probably separate the two methods so Local Server is always first and Chassis is the deprecated alternative.

If this change has already been agreed, please feel free to ignore these comments around that.

Reply:
Hmm, I guess this should have been a separate PR, but since it's on the same repo I decided to squeeze it in here. This change isn't part of the "upgrading to v12" documentation, but rather the general "Getting Started" guide ([public facing link](https://docs.altis-dxp.com/getting-started/)). I don't think we should be telling people about Chassis here, or that it has been deprecated. Would you agree?

There's a whole section in the upgrading to v12 doc about deprecating Chassis, which I think should be enough. It'll be need to be part of the beta and release announcements too. The Chassis documentation will still be available for v12, but will need an "is deprecated" PR on [its repo](https://github.com/humanmade/altis-local-chassis). Then in v13 it should completely be removed, except for references in the older upgrading docs.

Anyway, happy to reverse this change if you feel we should still mention it in the welcome guide.